app/models/miq_server.rb

```diff
@@ -218,12 +218,12 @@ def self.start
       update_server_config(cfg, :host, ipaddr)
     end
 
-    unless hostname.empty?
+    unless hostname.blank?
```

Comment:
I was thinking that we'd decide that separately.  For now this PR just stop things from working *at all*